sympy/simplify/radsimp.py

```diff
@@ -521,12 +521,34 @@ def collect_sqrt(expr, evaluate=None):
     return coeff*d
 
 
-def collect_const(expr, *vars, **kwargs):
+def collect_const(expr, *vars, Numbers=True):
```

Comment:
Python 2 does not allow named parameters like `Numbers=True` (`defparameter` in https://docs.python.org/2/reference/compound_stmts.html#function-definitions) after 'starred' parameters like `*vars`. The order should be changed. (Python 3 accepts both orders.)